ui/components/app/gas-customization/gas-slider/gas-slider.stories.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import GasSlider from '.';
+
+export default {
+  title: 'Components/UI/GasSlider',
+  id: __filename,
+  argTypes: {
+    onChange: {
+      action: 'onChange',
+    },
+    lowLabel: {
+      control: 'text',
+    },
+    highLabel: {
+      control: 'text',
+    },
+    value: {
+      control: 'number',
+    },
+    step: {
+      control: 'number',
+    },
+    max: {
+      control: 'number',
+    },
+    min: {
+      control: 'number',
+    },
+  },
+};
+
+export const DefaultStory = () => <GasSlider />;
+
+DefaultStory.storyName = 'Default';
```

Comment:
Adding missing story to help display changes

<img width="704" alt="Screen Shot 2022-07-20 at 9 56 13 AM" src="https://user-images.githubusercontent.com/8112138/180039840-b5e7706b-9833-4cfb-99d4-c2f030ee3695.png">